### PR TITLE
PSR-177

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Added the try before you buy task table view where users can try tasks before signing up.

### CODE REVIEW

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    ///
+    /// - parameter indexPath: from the table view
+    ///
+    /// - returns: the title for the task list row
+    ///
+    open func title(for indexPath: IndexPath) -> String? {
+        let taskId = tasks[indexPath.row]
+        switch taskId {
+        case .psoriasisDrawTask:
+            return "Finger Tapping"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
            return "Psoriasis Draw"
```

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        case .psoriasisAreaPhotoTask:
+            return "Phone Hold"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
            return "Psoriasis Area Photo"
```

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        case .jointCountingTask:
+            return "Finger-to-Nose"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
            return "Counting Painful Joints"
```

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        case .digitalJarOpenTask:
+            return "Dual Phone Hold"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
            return "Digital Jar Open"
```

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+        case .handImagingTask:
+            return "Digit Symbol Substitution"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
            return "Fingers Photo"
```

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+        case .footImagingTask:
+            return "Go-No-Go"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
            return "Toes Photo"
```

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+        case .walkingTask:
+            return "N-Back"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
            return "30 Second Walk"
```

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+//
+
+import Foundation
+import BridgeApp
+import MotorControl
+
+/// Subclass the schedule manager to set up a predicate to filter the schedules.
+public class TaskListScheduleManager : SBAScheduleManager {
+    
+    public let tasks: [RSDIdentifier] = [.psoriasisDrawTask, .psoriasisAreaPhotoTask, .jointCountingTask, .digitalJarOpenTask, .handImagingTask, .footImagingTask, .walkingTask, .mdJointCountingTask, .mdJointSwellingTask]
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
    public let tasks: [RSDIdentifier] = [.psoriasisDrawTask, .psoriasisAreaPhotoTask, .jointCountingTask, .digitalJarOpenTask, .handImagingTask, .footImagingTask, .walkingTask]
```

Comment (DanWebster) [reply Psorcast/Psorcast/TaskListScheduleManager.swift]: For doing try before you buy, we won't need to have the MD-based tasks

File: Psorcast/Psorcast/TaskListScheduleManager.swift [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+        case .handImagingTask:
+            return "Digit Symbol Substitution"
+        case .footImagingTask:
+            return "Go-No-Go"
+        case .walkingTask:
+            return "N-Back"
+        case .mdJointCountingTask:
+            return "Spatial Working Memory"
+        case .mdJointSwellingTask:
+            return "Spatial Working Memory"
```
Comment (DanWebster) [inline Psorcast/Psorcast/TaskListScheduleManager.swift]:
```suggestion
      
```

Comment (DanWebster) [reply Psorcast/Psorcast/TaskListScheduleManager.swift]: Similar to above, we won't need to have the MD-based tasks for the try before you buy

**Comment by dephillipsmichael** [comment]
Got the titles and subtitles fixed, good catch @DanWebster,

**Comment by DanWebster** [comment]
Thanks @dephillipsmichael for making the strings localizable; very forward-looking and I like it

**Pull request merged**